core/modules/date/date.admin.inc

```diff
@@ -207,7 +207,10 @@ function _date_field_instance_settings_form($field, $instance) {
     '#fieldset' => 'default_values',
   );
 
-  $description = t("Describe a time by reference to the current day, like '+90 days' (90 days from the day the field is created) or '+1 Saturday' (the next Saturday). See !strtotime for more details.", array('!strtotime' => l(t('strtotime'), 'http://www.php.net/manual/en/function.strtotime.php')));
+  $description = t(
+    "Describe a time by reference to the current day, like '+90 days' (90 days from the day the field is created) or '+1 Saturday' (the next Saturday). See !strtotime for more details.",
```

Comment:
Still wrong usage of line breaks here - please use them around array items, not inside the function.
```
  $description = t( "Describe a time by...", array(
```
No line break right after `t(`, but after `array(`.